### ISSUE TITLE
questions about sample

### ISSUE THREAD

**Issue by hsyfyc** [matsim-org/matsim-code-examples#816]
Hello, I have some questions about sample.
I want to run a simulation of a city with ten million people in urban area. While according to our investigation, there are in average four million people traveling in a single day. So if I create a scenario with 100k people, what values should I set for flowCapacityFactor and storageCapacityFactor? Now I set the value of these two parameters as "flowCapacityFactor=0.025" and "storageCapacityFactor=0.1", is this correct?
Then it comes to the public transpotation. I had read many issues on github, and had set the passengerCarEquivalents pce="0.05", PcuThresholdForFlowCapacityEasing=0.05. But I found that these two parameters seem not influence the supply of pt, so if the demand of people has been cut, should the supply of pt and taxi be cut too? If I need to do so, can you recommend proper way? Now I can only try to do so by sampling 2.5% taxi and cut the standing room and seats in pt vehicles.
Thanks!

**Comment by aayushtewari01** [comment]
@hsyfyc I would like to comment and discuss about the first part of the issue as I myself am trying to simulate such large scenarios.
As per the MATSim user book Page 38 FCF and SCF are to be the just as same as the sample population. So for 10% sample population SCF and FCF both are to be 0.1.

Some research papers have cited SCF to be more than FCF. SCF = FCF^0.75
You might also like to (or might have read the discussion) on issue #62  here.

For me, I couldn't find any proper guidelines or set rules about FCF and SCF and at times I have used FCF more than the sample population as well, as many agents remain stuck within the network (as per the leg histogram in output folder).  This may or may not be the issue with you. But in case you arrive at a conclusion, please mention here.

May I also ask what Count scale factor do you use for 100K population and does it provides a justifiable result.

Thank you

**Comment by hsyfyc** [comment]
@aayushtewari01
Thanks for your discussion. But I am still confused with FCF and SCF. Hope that somebody could solve this question.
For the count scale factor, I also want to know how to set it. Now I just keep the default value and change the link flow from linkstats after simulation manually.

**Comment by aayushtewari01** [comment]
@hsyfyc there are no concrete guidelines as I mentioned. I believe It also depends on how good your network capacity, freespeed is close to real-world network. If the network estimation is good then I think FCF and SCF can be just used as sample population factors as per Issue #62. 
So it is a kind of trial and error. In MATSim 2016 book chapter 90, SCF is 3. There are other research papers also where FCF and SCF are more than the sample populations


Asking about changing the link flow from linkstats manually, do you simply multiply all link count with a constant say 10 for a sample population of 10%. In which case how good is your results?

**Comment by hsyfyc** [comment]
@aayushtewari01 yes, but the result analysis is my teammate's work, so I am sorry that I cannot tell you the details about the results.
